README.md

```diff
@@ -95,4 +95,15 @@ There are separate lists for [Open Source by Higher Education](https://github.co
 
 * [OpenGovernment](https://github.com/opengovernment) (this should be probably moved to non-profits)
 
-
+* [code\.gov](https://github.com/GSA/code-gov)
+    * This initiative includes most US government opensource repos. I will pick some interesting ones here. 
```

Comment:
here the link should go to https://github.com/GSA and IMHO there is no need for the escaping in the square brackets.